frontstage/controllers/party_controller.py

```diff
@@ -145,6 +145,18 @@ def resend_verification_email(party_id):
     logger.debug('Successfully re-sent verification email', party_id=party_id)
 
 
+def resend_verification_email_expired_token(token):
+    logger.debug('Re-sending verification email', token=token)
```

Comment:
@gavinleeedwards Just a note on this and a lot of the controllers in this file, they are all quite similar, might be worth adding a tech debt ticket to look to refactor and move common bits to private functions. Just a thought

Reply:
Could be a good idea @shahi645 but worth noting that we used to have a similar requests wrapper which did that. 
It had the exception handling/basic auth stuff handled inside it. In general though we found that it was just an extra thing to worry about as we had to make modifications to it every time a new scenario came up for the request (new ways of handling errors/different types of request/sending different types of data)